build.sh

```diff
@@ -11,9 +11,11 @@ SCRIPT_DIR="$(readlink -f "$(dirname -- "${BASH_SOURCE[0]}")")"
 BUILD_TYPE="${BUILD_TYPE:-Release}"
 INSTALL_PREFIX="${INSTALL_PREFIX:-/usr/local}"
 
+# TODO remove ENABLE_ZERO_COPY
 cmake -B "${SCRIPT_DIR}/out" \
     -DCMAKE_BUILD_TYPE="${BUILD_TYPE}" \
     -DCMAKE_INSTALL_PREFIX="${INSTALL_PREFIX}" \
+    -DENABLE_ZERO_COPY=OFF \
```

Comment:
why disable zero copy?
Rest seems rather ok

Reply:
It causes data corruption. I don't know if the fault is on the MCM side or MTL, but it is highly possible that the issue is in MTL. More investigation is needed, and disabling zero copy is only a temporary fix.

Reply:
I removed this part as it is fixed in different PR